credentials/apps/edx_credentials_extensions/edly_credentials_app/api/serializers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Serializers for edly_api.
+"""
+from django.contrib.sites.models import Site
+
+from rest_framework import serializers
+
+from credentials.apps.credentials.models import ProgramCertificate, Signatory
+from credentials.apps.edx_credentials_extensions.edly_credentials_app.api.fields import Base64ImageField
+
+
+class SignatorySerializer(serializers.ModelSerializer):
+    """
+    Serializer for Signatory.
+    """
+    image = Base64ImageField(max_length=None, use_url=True)
+
+    class Meta:
+        model = Signatory
+        fields = ['id', 'name', 'title', 'image']
+
+
+class ProgramCertificateConfigurationSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Program Certificate Configuration.
+    """
+    site = serializers.SlugRelatedField(slug_field='domain', queryset=Site.objects.all())
+    signatories = SignatorySerializer(many=True)
+
+    class Meta:
+        model = ProgramCertificate
+        fields = [
+            'site', 'is_active', 'signatories', 'program_uuid', 'use_org_name',
+            'include_hours_of_effort', 'language',
+        ]
+
+    def create(self, validated_data):
+        signatories = []
+        for signatory in validated_data.pop('signatories', []):
+            signatories.append(Signatory.objects.create(**signatory))
+
+        program_certificate_configuration = ProgramCertificate(**validated_data)
+        program_certificate_configuration.save()
+
```

Comment:
nit: remove unnecessary newlines. We usually add newlines after de-indenting a piece of code

Reply:
Done